webapp/src/index.css

```diff
@@ -20,6 +20,13 @@
   --primary-font: 'DM Sans', sans-serif;
 }
 
+/* Styles for viewports wider than 768 pixels. */
+@media (min-width: 768px) {
+  :root {
+    font-size: 120%;
```

Comment:
I don't understand this. Could you elaborate, please?

I think the way it works now takes into account the font size settings of the browser.

For example, if the browser has 16px as the default font size, on mobile, a font size of 2rem is 32px, and on the desktop is 16px * 1.2 * 2 = 38.4px~.

If the browser has 24px or other values as default, this considers that.
For example, a font size of 2rem is 48px on mobile and 24px * 1.2 * 2 = 57.6 on desktop.

Setting the font size to a [pixel](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#pixels) value in the `:root` element ignores the browser settings.

Reply:
Yes, and because we depend on browser root font size I think we should have controlled ours (as fair as I've seen we don't have this yet but maybe I'm wrong)

What I mean is that we should define our base root font size and just re-define it in the media-query explicitly. Despite I was thinking in defining it explicitly, this is in pixels, a percentage seems a good option too

Reply:
I've added `font-size: 100%` by default to account for the browser settings and `120%` for screen size greater than 768px (we could change this).

I think setting it to pixels would also work and be responsive. Setting it to a percentage is responsive and accessible.

Most users use the default font size, which on most browsers is 16px. And those who do change it usually do it for accessibility reasons.

What do we think about this?

Reply:
If you are sure about all browsers set 16px by default (**including mobile ones**) I agree with setting the default font to 100%. 

I don't see what are we loosing setting it directly to 16px, what are the disadvantages? In the other side, I see a clear disadvantage with the % one, and it's I am not 100% sure we will always be getting 16px as `root` by default in **all devices**. With the 16px option I am 100% sure :smile:

Reply:
I see your point :+1:. I've changed it.